src/pages/Results.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, useEffect, useState, useContext } from "react";
+import { FhirContext } from "../components/fhir/FhirContext";
+
+import LoadingSpinner from "../components/UI/LoadingSpinner";
+import ModalWrapper from "../components/UI/ModalWrapper";
+import { ModalState } from "../components/UI/ModalWrapper";
+import ResultsList from "../components/results-list/ResultsList";
+
+const FHIR_URL = process.env.REACT_APP_FHIR_URL;
+
+type patientResult = {
+  firstName: string;
+  lastName: string;
+  patientId: string;
+  observations: { [key: string]: any }[]; // using any since it is a deeply nested object from an external api}
```

Comment:
I think the @smile-cdr types should help you out here
```suggestion
  observations: Observation[];
```